src/Components/App/App.js

```diff
@@ -1,11 +1,39 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import './App.scss';
+import Nav from '../Nav/Nav.js';
+import Login from '../Login/Login.js';
 
-function App() {
-  return (
 
-  );
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
```

Comment:
Nice job setting up the class component here. I think we are off to a great start setting these up and updating the state in App for when the user logs in. I think we can then pass this.state.userInfo into the Nav component once we start working on that functionality